test/base/test_store.rb

```diff
@@ -32,4 +32,16 @@ def test_delete_and_reload_on_error
     store.load
     assert_equal(nil, store.data)
   end
+
+  def test_tmp_path_with_nil_env
+    tmp_path_for_checksum = Nanoc::Int::Store.tmp_path_for(nil, 'checksum')
+    assert_equal('tmp/checksum', tmp_path_for_checksum)
+  end
+
+  def test_tmp_path_with_test_env
+    tmp_path_for_checksum = Nanoc::Int::Store.tmp_path_for('test', 'checksum')
+    tmp_path_for_dependencies = Nanoc::Int::Store.tmp_path_for(:test, 'dependencies')
```

Comment:
I’d remove support for symbols, and only use strings instead (there’s no use case for symbols).